### PR TITLE
Adding single page bikeshed wrapper build of core spec.

### PR DESCRIPTION
**Pull request by flagxor**
This produces a single page bikeshed wrapper around the core spec.
A bunch of tex seems to croak with MathJax if it's not broken apart, so did that.

Tab, I had to work around |foo| getting swallowed by bikeshed into <var>foo</var>, does that rise to the level of a bug?

Eric, the resulting doc nominally looks ok, but perhaps we could just incorporate the appropriate text from within Spinx ?

### CODE REVIEW

File: document/core/conf.py [hunk 1]
```diff
@@ -129,24 +129,28 @@
...
-  'sidebar_width': '260px',
-  'sidebar_collapse': True,
-  'show_powered_by': False,
-  'extra_nav_links': {
-    'Index': 'BASEDIR/genindex.html',
-    'Download as PDF': 'BASEDIR/_download/' + name + '.pdf'
-  },
+  'nosidebar': True,
+
+#  'logo': logo,
```
[the remark this answers is not in the document]
Comment (flagxor) [reply document/core/conf.py]: Ah, yeah, not thinking.
Splitting out into an alternate config.

**Comment by flagxor** [comment]
Ok, this seems to be able to build everything.
See result here:
http://flagxor.github.io/spec/

PTAL

**Comment by littledan** [comment]
@flagxor Great job on this document. It's fast to load and comfortable to read!

File: document/core/katex_fix.patch [hunk 2]
```diff
@@ -0,0 +1,29 @@
+diff --git a/dist/katex.css b/dist/katex.css
```
Comment (rossberg) [inline document/core/katex_fix.patch]: Can you move this file to core/util?

File: document/core/mathjax2katex.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env python
```
Comment (rossberg) [inline document/core/mathjax2katex.py]: Also move to util?

**Comment by flagxor** [comment]
PTAL
Was able to drop the rst changes as katex doesn't choke (only mathjax).
Did each of the comments.

File: .gitmodules [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[submodule "document/core/katex"]
```
[the remark this answers is not in the document]
Comment (rossberg) [reply .gitmodules]: Don't you need to adjust the above path as well?

File: document/core/bikeshed/conf.py [hunk 5]
```diff
@@ -0,0 +1,470 @@
+# -*- coding: utf-8 -*-
```
Comment (rossberg) [inline document/core/bikeshed/conf.py]: Nit: move this to /util as well.

Comment (flagxor) [reply document/core/bikeshed/conf.py]: Moved just the bikeshed one for now.
Probably should consider the other one too at some point. Although it's kind of a gray area whether the config is part of the document or generic.

**Pull request merged**